lib/controller/fiscaat/BeheerCiviProductenController.class.php

```diff
@@ -79,6 +81,16 @@ public function POST_toevoegen() {
 		$this->view = $form;
 	}
 
+	/**
+	 * @throws \CsrDelft\common\CsrGebruikerException
+	 */
+	public function POST_nieuw() {
```

Comment:
Deze code hoort in de PeilingenController denk ik?

Reply:
😲